platforms/win32/vm/sqWin32Window.c

```diff
@@ -625,6 +625,10 @@ MainWndProcW(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
   case WM_KILLFOCUS:
     fHasFocus = 0;
     return DefWindowProcW(hwnd,message,wParam,lParam);
+
+  case WM_TIMECHANGE:
+    resyncSystemTime();
+
```

Comment:
For what it is worth, wine has no default reaction to WM_TIMECHANGE https://github.com/wine-mirror/wine/blob/master/dlls/user32/defwnd.c

One might speculate whether Windows will in the future at some point have a meaningful default reaction to that message (which would speak for DefWindowProc). I doubt it will (which speaks for break).

But anyway I would not keep the fall-through because otherwise the next person to add a message here might overlook it and introduce unexpected behavior whenever WM_TIMECHANGE comes in.